app/src/main/java/de/westnordost/streetcomplete/util/ktx/Context.kt

```diff
@@ -51,16 +50,13 @@ fun Context.toast(@StringRes resId: Int, duration: Int = Toast.LENGTH_SHORT) {
 fun Context.hasPermission(permission: String): Boolean =
     ContextCompat.checkSelfPermission(this, permission) == PackageManager.PERMISSION_GRANTED
 
-fun View.showKeyboard(): Boolean? =
-    context?.inputMethodManager?.showSoftInput(this, SHOW_IMPLICIT)
+fun View.showKeyboard() = SoftwareKeyboardControllerCompat(this).show()
```

Comment:
Would you specify the return type for this function and hideKeyboard explicitly?

Reply:
It doesn't return anything.

Reply:
Yes, so it should be `: Unit`

If not specified, the return type is set automatically to whatever the return type of the expression on the right is. For a reader without an IDE that can help him, it is then not obvious what the return type is.